src/js/views/maps/viewfinder/ViewfinderView.js

```diff
@@ -96,6 +131,9 @@ define(
         this.el.innerHTML = _.template(Template)(this.templateVars);
 
         this.renderSearchView();
+        if (this.model.get('showZoomPresets')) {
```

Comment:
just noticed that this shows an empty drop down when zoomPresets are not defined. we should probably add a zoomPresets check here as well.

Reply:
Can do!